cmd/version-control/main.go

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"log"
+	"net/http"
+	"strings"
+
+	"storj.io/storj/internal/version"
+)
+
+var (
+	ver []version.Info
+)
+
+func handleGet(w http.ResponseWriter, r *http.Request) {
+	var xfor string
+
+	// Only handle GET Requests
+	if r.Method == "GET" {
+		if xfor = r.Header.Get("X-Forwarded-For"); xfor == "" {
+			xfor = r.RemoteAddr
+		}
+		log.Printf("Request from: %s for %s", r.RemoteAddr, xfor)
+
+		w.Header().Set("Content-Type", "application/json")
+		response, err := json.Marshal(ver)
```

Comment:
If the version isn't ever going to change, and we want to go fast, we should only Marshal this once at start. json and go is notoriously slow.

Reply:
that's a fair point. :+1:

Reply:
Done